modules/apps/polls/polls-web/src/main/java/com/liferay/polls/web/internal/portlet/PollsPortlet.java

```diff
@@ -29,25 +39,19 @@
 @Component(
 	immediate = true,
 	property = {
-		"com.liferay.portlet.add-default-resource=true",
 		"com.liferay.portlet.css-class-wrapper=portlet-polls",
 		"com.liferay.portlet.display-category=category.hidden",
 		"com.liferay.portlet.header-portlet-css=/css/main.css",
 		"com.liferay.portlet.icon=/icons/polls.png",
+		"com.liferay.portlet.instanceable=false",
 		"com.liferay.portlet.preferences-owned-by-group=true",
 		"com.liferay.portlet.preferences-unique-per-layout=false",
 		"com.liferay.portlet.private-request-attributes=false",
 		"com.liferay.portlet.private-session-attributes=false",
 		"com.liferay.portlet.render-weight=50",
-		"com.liferay.portlet.scopeable=true",
-		"com.liferay.portlet.struts-path=polls",
 		"com.liferay.portlet.use-default-template=true",
 		"javax.portlet.display-name=Polls", "javax.portlet.expiration-cache=0",
-		"javax.portlet.info.keywords=Polls",
-		"javax.portlet.info.short-title=Polls",
-		"javax.portlet.info.title=Polls",
-		"javax.portlet.init-param.template-path=/META-INF/resources/",
-		"javax.portlet.init-param.view-action=/polls/view",
+		"javax.portlet.init-param.template-path=/polls/",
```

Comment:
Hi @natocesarrego. Sorry about that. Because I was using `DDMFormAdminPortlet.java` as a reference, those properties ended up to be removed, but they don't prevent my fix to work. Thanks!

Reply:
No problem, @carolmariaabb. 🙂 Can you please re-send without removing these unnecessary properties? Thanks.